src/hooks_test.rs

```diff
@@ -185,7 +185,45 @@ mod create_hook_files_tests {
             assert_eq!(true, make_executable);
             Ok(())
         };
-        let result = create_hook_files(write_file, root_dir, git_hooks);
+        let result = create_hook_files(write_file, root_dir, git_hooks, &vec![]);
+        assert_eq!(result, Ok(()));
+    }
+
+    #[test]
+    fn creates_skipped_hook() {
```

Comment:
thanks for the updates! think we'll want to update the function name to clarify the intent of this test, which is to validate that we _don't_ attempt to write the hook file for a hook that's supposed to be skipped

```suggestion
    fn does_not_create_skipped_hook() {
```